spec/support/shared_examples/validators/shared_examples_for_advocate_claim_validators.rb

```diff
@@ -97,12 +97,227 @@
 
 RSpec.shared_examples 'advocate claim supplier number' do
   context 'supplier_number' do
-      # NOTE: In reality supplier number is derived from external_user which in turn is validated in any event
-      let(:advocate) { build(:external_user, :advocate, supplier_number: '9G606X') }
+    # NOTE: In reality supplier number is derived from external_user which in turn is validated in any event
+    let(:advocate) { build(:external_user, :advocate, supplier_number: '9G606X') }
+
+    it 'should error when the supplier number does not match pattern' do
+      claim.external_user = advocate
+      should_error_with(claim, :supplier_number, 'invalid')
+    end
+  end
+end
+
+RSpec.shared_context 'seeded fee schemes' do
```

Comment:
cool. **not a blocker** but for contexts i would use a command type statement - seed-fee-schemes - without  or without the hyphens

Reply:
👍 madre suggested change "seed-fee-schemes"